### ISSUE TITLE
Graylog 3.1.2 migration failed for `Graylog Events` after change replicas count

### ISSUE THREAD

**Issue by R4scal** [Graylog2/graylog2-server#6507]
Hi
## Steps to Reproduce (for bugs)
1. Install graylog 3.1.2
2. Update index setttings for `Graylog Events` indexset:
repicas: 0 -> 1
rotation period: P1M -> P7D
3. Restart graylog-server
4. Errors in logs:
```
2019-09-30T14:03:45.469Z ERROR [ConfigurationManagementPeriodical] Error while running migration <V20190705071400_AddEventIndexSetsMigration{2019-07-05T07:14:00Z}>
java.lang.IllegalStateException: Index prefix conflict: a non-events index-set with prefix <gl-events> already exists. Configure a different <default_events_index_prefix> value in the server config file.
	at org.graylog2.migrations.V20190705071400_AddEventIndexSetsMigration.checkIndexPrefixConflicts(V20190705071400_AddEventIndexSetsMigration.java:130) ~[graylog.jar:?]
	at org.graylog2.migrations.V20190705071400_AddEventIndexSetsMigration.ensureEventsStreamAndIndexSet(V20190705071400_AddEventIndexSetsMigration.java:111) ~[graylog.jar:?]
	at org.graylog2.migrations.V20190705071400_AddEventIndexSetsMigration.upgrade(V20190705071400_AddEventIndexSetsMigration.java:84) ~[graylog.jar:?]
	at org.graylog2.periodical.ConfigurationManagementPeriodical.doRun(ConfigurationManagementPeriodical.java:43) [graylog.jar:?]
	at org.graylog2.plugin.periodical.Periodical.run(Periodical.java:77) [graylog.jar:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_222]
```

## Your Environment

* Graylog Version: 3.1.2
* Elasticsearch Version: 6.8.1
* MongoDB Version: 4.2
* Operating System: Ubuntu 18.04

**Comment by mj84** [comment]
I can confirm this on a fresh install

**Comment by bernd** [comment]
@R4scal Thank you for the report. This will be fixed in the upcoming 3.1.3 release.
